domain-models-runtime/src/test/java/org/folio/rest/persist/PostgresClientITBase.java

```diff
@@ -41,6 +42,24 @@ protected static void setUpClass(TestContext context) throws Exception {
         "CREATE ROLE " + schema + " PASSWORD '" + tenant + "' NOSUPERUSER NOCREATEDB INHERIT LOGIN",
         "CREATE SCHEMA " + schema + " AUTHORIZATION " + schema,
         "GRANT ALL PRIVILEGES ON SCHEMA " + schema + " TO " + schema);
+    loadFuncs(context, PostgresClient.getInstance(vertx));
+  }
+
+  private static void loadFuncs(TestContext context, PostgresClient postgresClient) {
+    Async async = context.async();
+    try {
+      String sql = IOUtils.toString(
+        PostgresClientITBase.class.getClassLoader().getResourceAsStream("templates/db_scripts/funcs.sql"), "UTF-8");
```

Comment:
Can this copy & paste been avoided?

funcs.sql is a copy of general_functions.ftl any can become out of sync. I suggest to delete funcs.sql and to read general_functions.ftl instead. Use replaceAll to replace the variables.

Reply:
Yeah.. It's annoying.. Violation of DRY. I'll see if I can fix that.